pyccel/codegen/cwrapper.py

```diff
@@ -59,7 +60,7 @@ def create_c_setup(mod_name,
     else:
         code += "\n"
 
-    wrapper_file = "[ r'{0}' ]".format(wrapper_file)
+    wrapper_file = "[ r'{0}' , r'cwrapper/cwrapper.c']".format(wrapper_file) #TODO find another way
```

Comment:
If this file is compiled every time then you need to add it to the file locking system [here](https://github.com/pyccel/pyccel/blob/master/pyccel/codegen/python_wrapper.py#L135-L143) to prevent 2 threads trying to use it at the same time (this is the cause of the current failures)